wagtailio/static/sass/components/_comparison-table.scss

```diff
@@ -0,0 +1,42 @@
+.comparison-table {
+    align-items: start;
+
+    td:first-child {
+        font-weight: $weight--bold;
+    }
+
+    tbody {
+        grid-column: 2 / span 1;
```

Comment:
suggestion: update this to `grid-column: 2 / span 2;` to span the 2 content columns we have on mobile